src/app/common/directives/oc-file-upload/js/oc-file-upload.js

```diff
@@ -24,14 +24,14 @@ function ordercloudFileUpload($uibModal, $ocFiles, ocFiles, ocConfirm) {
         })();
 
         function initModelValue() {
-            if (!scope.fileUploadOptions.multiple) {
-                if (!scope.fileUploadModel[scope.fileUploadOptions.keyname] || scope.fileUploadModel[scope.fileUploadOptions.keyname].constructor !== Object) 
-                    scope.fileUploadModel[scope.fileUploadOptions.keyname] = {};
-            }
-            else {
-                if (!scope.fileUploadModel[scope.fileUploadOptions.keyname] || scope.fileUploadModel[scope.fileUploadOptions.keyname].constructor !== Array) {
-                    scope.fileUploadModel[scope.fileUploadOptions.keyname] = [];
-                }
+            if (scope.fileUploadModel) scope.model = scope.fileUploadModel;
+            scope.model ? scope.fileUploadModel = angular.copy(scope.model) : scope.fileUploadModel = {};
```

Comment:
This could be:

scope.fileUploadModel = angular.copy(scope.model ? scope.model : {});

but not a big deal.